manager/api/views.py

```diff
@@ -547,7 +562,10 @@ def salinfo_topic_names(request):
     query = ""
     if "categories" in request.query_params:
         query = "?categories=" + request.query_params["categories"]
-    url = f"http://{os.environ.get('COMMANDER_HOSTNAME')}:{os.environ.get('COMMANDER_PORT')}/salinfo/topic-names{query}"
+    url = (
+        f"http://{os.environ.get('COMMANDER_HOSTNAME')}:"
```

Comment:
Is there a way to add https somehow? Can that be an ENV variable?

Reply:
This is a great suggestion @pkubanek, although I think I'll leave this for another ticket as it would mean a refactor not realted to this specific PR. Created: [DM-42782](https://jira.lsstcorp.org/browse/DM-42782).